cpu/kinetis/dist/check-fcfield.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+#
+# Flash configuration field check script for Freescale Kinetis MCUs.
+#
+# This script is supposed to be called from RIOTs
+# unified OpenOCD script (dist/tools/openocd/openocd.sh).
+#
+# Syntax: check-fcfield.sh $FLASHFILE
+#
+# @author       Jonas Remmert <j.remmert@phytec.de>
+# @author       Johann Fischer <j.fischer@phytec.de>
+# @author       Joakim Nohlgård <joakim.nohlgard@eistec.se>
+# @author       Francisco Molina <francisco.molina@inria.fr>
+
+if [ $(printf '%d' "${IMAGE_OFFSET}") -gt 1040 ] ; then
+    echo "WARN: we don't check the fcfield value if flashing at \$IMAGE_OFFSET > 0x410"
+    exit 0
+fi
+
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 FLASHFILE"
+    echo "Checks the flash configuration field protection bits to avoid flashing a locked image to a Kinetis MCU (protection against accidental bricking)."
+    exit 2
+fi
+
+FLASHFILE="$1"
+
+FCFIELD_START='0x400'
+FCFIELD_END='0x410'
+FCFIELD_AWK_REGEX='^ 0400 '
+
+if [ ${FLASHFILE##*.} = bin ]; then
+    OBJDUMP=$(arm-none-eabi-objdump --start-address=${FCFIELD_START} --stop-address=${FCFIELD_END} -bbinary -marm ${FLASHFILE} -s)
```

Comment:
Better not use `OBJDUMP` as variable name here. In the build system `OBJDUMP` is the name of the tool, and I have a pull request to use `OBJDUMP` from the build system in this file :D (I will do it after this one).

I personally tend to not put commands output into a variable when possible (from this source, in French https://linuxfr.org/news/revue-des-techniques-de-programmation-en-shell). So would more refactor this in a function and use the output with `get_fc_field "${FLASHFILE}" | awk`.
The fact that you need to do `printf %s ${OBJDUMP}` somehow shows the pattern issue.

So at least rename it, and maybe put it in a function

Reply:
Woops, will move to the dedicated pull request :)